ArduPlane/GCS_Mavlink.cpp

```diff
@@ -1018,8 +1018,22 @@ MAV_RESULT GCS_MAVLINK_Plane::handle_command_long_packet(const mavlink_command_l
 
     case MAV_CMD_DO_AUTOTUNE_ENABLE:
         // param1 : enable/disable
-        plane.autotune_enable(!is_zero(packet.param1));
-        return MAV_RESULT_ACCEPTED;
+        // param2 : axis to tune
+        if ((u_int8_t)packet.param2 == 0) {
+            plane.autotune_enable(!is_zero(packet.param1));
+            return MAV_RESULT_ACCEPTED;
+        } else if ((u_int8_t)packet.param2 & AP_AutoTune::AUTOTUNE_ROLL) {
+            plane.autotune_enable(!is_zero(packet.param1), AP_AutoTune::AUTOTUNE_ROLL);
+            return MAV_RESULT_ACCEPTED;
+        } else if ((u_int8_t)packet.param2 & AP_AutoTune::AUTOTUNE_PITCH) {
```

Comment:
This construct means that you'll only ever tune one axis at a time.... not correct :-)